Finder/TransformedFinder.php

```diff
@@ -97,6 +104,17 @@ public function findPaginated($query, $options = array())
 
         return new Pagerfanta(new FantaPaginatorAdapter($paginatorAdapter));
     }
+    
+    /**
+     * {@inheritdoc}
+     */
+    public function findRawPaginated($query, $options = array())
+    {
+        $queryObject = Query::create($query);
+        $paginatorAdapter = $this->createRawPaginatorAdapter($queryObject, $options);
+
+        return new Pagerfanta(new FantaPaginatorAdapter($paginatorAdapter));
```

Comment:
I'm not entirely sure if this belongs in `TransformedFinder` as it doesn't really transforms something?